src/components/data-exploration/differential-expression-tool/LaunchPathwayAnalysisModal.jsx

```diff
@@ -50,11 +87,11 @@ const LaunchPathwayAnalysisModal = (props) => {
       pushNotificationMessage('error', 'Failed launching pathway analysis');
       console.error('Error launching pathway analysis', error);
     } finally {
-      setWaitingForExternalService(false);
+      setLaunchingPathwayAnalysis(false);
     }
   };
 
-  const canLaunchService = () => !waitingForExternalService && species;
+  const canLaunchService = () => !gettingBackgroundGenes && species;
```

Comment:
Before this was dependent on `waitingForExternalService` (now renamed to `launchingPathwayAnalysis`), now it is determined by `gettingBackgroundGenes` which is a completely different thing. 
I don't know if `gettingBackgroundGenes` should affect whether we can start a pathway analysis or not, since they are independent things.

Reply:
No, I don't think so. I think I mixed up during renaming. I'll switch it back.